tsconfig.json

```diff
@@ -93,16 +93,18 @@
     // "allowUnusedLabels": true,                        /* Disable error reporting for unused labels. */
     // "allowUnreachableCode": true,                     /* Disable error reporting for unreachable code. */
     /* Completeness */
-    // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
-    // "skipLibCheck": true                              /* Skip type checking all .d.ts files. */
+    "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
+    "skipLibCheck": true                              /* Skip type checking all .d.ts files. */
```

Comment:
Is this needed, given that you're excluding `dist` directories? It would be better to leave it false if possible. Ah, I see, it turns off checking of `node_modules`, which currently fails (from missing `@types` perhaps?). Still, seems like this turns off a lot of actual type checking... though we could leave it for later. Same for `skipDefaultLibCheck` which seems risky.

Reply:
This is needed because `unplugin`'s dts imports `webpack`, `vite`, `rollup`, `rspack`, and `esbuild` out of which we don't have `webpack` and `rspack` required in Civet, and `@typescript/vfs` imports `lz-string`. So when typescript does type checking it errors out on these imports. We could also install types for these packages, or add them to dev deps instead

Reply:
Adding dev dependencies for types sounds good.